<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>431</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>100</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>729</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="imge_tab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <attribute name="icon">
       <iconset>
        <normaloff>image.png</normaloff>image.png</iconset>
      </attribute>
      <attribute name="title">
       <string>IMAGE</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="path_img_line"/>
         </item>
         <item>
          <widget class="QPushButton" name="browse_img_b">
           <property name="styleSheet">
            <string notr="true">background :rgb(173, 180, 188)</string>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="prepare_img_b">
           <property name="styleSheet">
            <string notr="true">background :rgb(173, 180, 188)
</string>
           </property>
           <property name="text">
            <string>prepare Images</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QComboBox" name="algo_img_Box">
             <item>
              <property name="text">
               <string>Mean Color</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Histogram</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Color Layout</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="analy_img_btn">
             <property name="styleSheet">
              <string notr="true">background :rgb(173, 180, 188)</string>
             </property>
             <property name="text">
              <string>analyze</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="feat_img_line"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="img_list"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="vedio_tab">
      <attribute name="icon">
       <iconset>
        <normaloff>video.ico</normaloff>video.ico</iconset>
      </attribute>
      <attribute name="title">
       <string>VEDIO</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="path_ved_line"/>
         </item>
         <item>
          <widget class="QPushButton" name="browse_ved_b">
           <property name="styleSheet">
            <string notr="true">background :rgb(173, 180, 188)</string>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="yolo_vid_b">
           <property name="styleSheet">
            <string notr="true">background :rgb(173, 180, 188)</string>
           </property>
           <property name="text">
            <string>prepare videos</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QPushButton" name="start_ved_b">
           <property name="styleSheet">
            <string notr="true">background :rgb(173, 180, 188)</string>
           </property>
           <property name="text">
            <string>analyze</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="feat_ved_line"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
